cinex/Models/UpcomingModel.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace cinex.Models
+{
+    public class UpcomingModel
+    {
+        [JsonProperty(PropertyName = "vote_count")]
+        public int VoteCount { get; set; }
+
+        public int Id { get; set; }
+
+        public bool Video { get; set; }
+
+        [JsonProperty(PropertyName = "vote_average")]
+        public double VoteAverage { get; set; }
+
+        public string Title { get; set; }
+
+        public double Popularity { get; set; }
+
+        [JsonProperty(PropertyName = "poster_path")]
+        public string PosterPath { get; set; }
+
+        public string PosterPath_W200 => $"https://image.tmdb.org/t/p/w200/{PosterPath}";
+
+        [JsonProperty(PropertyName = "original_language")]
+        public string OriginalLanguage { get; set; }
+
+        [JsonProperty(PropertyName = "original_title")]
```

Comment:
You can use a `SnakeCaseNamingStrategy` to avoid having to annotate all properties :)

Reply:
I didn't know this Strategy. Implemented =)